src/main/resources/config/fields.yaml

```diff
@@ -1,533 +1,575 @@
-  entry:
-    cardinality: "1"
-    datatype: "datetime"
-    field: "end-date"
-    phase: "alpha"
-    register: ""
-    text: "End datetime for the applicability of a register entry."
-- serial-number: 12
-  hash: "6b6dba22e7e8e9103924059a8b77d03f94a1fbdd"
-  entry:
-    cardinality: "1"
-    datatype: "string"
-    field: "email"
-    phase: "alpha"
-    register: ""
-    text: "An email address for contact."
-- serial-number: 11
-  hash: "dd22f2db37c73d2dc1d99d22c0282cfec14308c4"
-  entry:
-    cardinality: "1"
-    datatype: "string"
-    field: "datatype"
-    phase: "alpha"
-    register: "datatype"
-    text: "The data type for constraining a field value."
-- serial-number: 10
-  hash: "0e00ce00a433bc68b9c30338fd2c2d71c68f7f08"
-  entry:
-    cardinality: "1"
-    datatype: "url"
-    field: "crest"
-    phase: "alpha"
-    register: ""
-    text: "Official crest for a government body."
-- serial-number: 9
-  hash: "12ee27755b0118bc391b18d804ec21abb51a9479"
-  entry:
-    cardinality: "1"
-    datatype: "string"
-    field: "country"
-    phase: "alpha"
-    register: "country"
-    text: "ISO 3166-2 two letter code for a country."
-- serial-number: 8
-  hash: "15000add6b8a0625abd14fa3f5c39dc54abe92aa"
-  entry:
-    cardinality: "1"
-    datatype: "text"
-    field: "copyright"
-    phase: "alpha"
-    register: ""
-    text: "Copyright for the data in the register."
-- serial-number: 7
-  hash: "15955914ceb90a824c2fe6e2432455814169268a"
-  entry:
-    cardinality: "1"
-    datatype: "string"
-    field: "company"
-    phase: "alpha"
-    register: "company"
-    text: "A UK registered company."
-- serial-number: 6
-  hash: "dace043fab243727edac7789d213df708dd7352f"
-  entry:
-    cardinality: "1"
-    datatype: "string"
-    field: "company-status"
-    phase: "alpha"
-    register: ""
-    text: "The status of a UK company."
-- serial-number: 5
-  hash: "aca249580d9a92023b207f3c74ea54b8e038e67e"
-  entry:
-    cardinality: "1"
-    datatype: "string"
-    field: "citizen-names"
-    phase: "alpha"
-    register: ""
-    text: "Name for the citzens of a country."
-- serial-number: 4
-  hash: "4de105d95a62b4cbcbaecbbc592ba52a745acb51"
-  entry:
-    cardinality: "1"
-    datatype: "string"
-    field: "cardinality"
-    phase: "alpha"
-    register: ""
-    text: "Number of elements of the set (1 or n)."
-- serial-number: 2
-  hash: "bf4172b99e36c850d3ad391e90462107d9603980"
-  entry:
-    cardinality: "1"
-    datatype: "string"
-    field: "area"
-    phase: "alpha"
-    register: ""
-    text: "The administrative area of an address."
-- serial-number: 1
-  hash: "91fa9952f411c9f4cd2067cbbd233b3d51449fdc"
-  entry:
-    cardinality: "1"
-    datatype: "string"
-    field: "address"
-    phase: "alpha"
-    register: "address"
-    text: "A place in the UK with a postal address."
+country:
+  entry-number: "9"
+  item-hash: "sha-256:a77c35a853ea8adce1922cadd664fe6548a7b34e322e53f8d28517ee22138ee5"
```

Comment:
I think this was copied in presentation repo from the old `prod` environment and then copied in here from presentation.